bin/build

```diff
@@ -6,7 +6,7 @@ set -ex
 
 . bin/build_utils
 
-CURRENT_DIR=$($(dirname $0)/abspath)
+CURRENT_DIR=$("$(dirname "$0")/abspath")
```

Comment:
wdyt about setting `prefix=$(dirname "$0")` before this line? having a command inside a command makes it harder to read imo

Reply:
all this stuff will be changed when we do:

> Add ticket to cleanup the way we’re determining path per the discussion in about bash today.

ie, the way we're doing abspath is wrong anyway (we shouldn't be using `$0`, eg), so i'm not sure this worth is fixing since all this code will go away when that's fixed.  i will if you feel strongly.